src/item.cpp

```diff
@@ -5531,9 +5531,6 @@ bool item::detonate( const tripoint &p, std::vector<item> &drops )
     } else if( type->ammo && ( type->ammo->special_cookoff || type->ammo->cookoff ) ) {
         long charges_remaining = charges;
         const long rounds_exploded = rng( 1, charges_remaining );
-        // Yank the exploding item off the map for the duration of the explosion
-        // so it doesn't blow itself up.
-        item temp_item = *this;
```

Comment:
I'm not sure what this is doing, but this code has broken a lot in the past, please remove this and put the change it it's own PR.